chrome/content/zotero/xpcom/collectionTreeView.js

```diff
@@ -662,6 +662,9 @@ Zotero.CollectionTreeView.prototype.getImageSrc = function(row, col)
 	
 	switch (collectionType) {
 		case 'library':
+			if(! treeRow.editable) {
```

Comment:
Style nit: For all of these, space after `if` (keyword, not function), no space after `!`.